Source/Core/DolphinQt/SkylanderPortal/SkylanderPortalWindow.cpp

```diff
@@ -321,30 +321,36 @@ QVBoxLayout* SkylanderPortalWindow::CreateFinderLayout()
   m_element_filter[0]->setChecked(true);
   // i18n: One of the elements in the Skylanders games. Japanese: まほう. For official translations
   // in other languages, check the SuperChargers manual at https://support.activision.com/manuals
-  m_element_filter[1]->setText(tr("Magic"));
+  m_element_filter[GetElementID(IOS::HLE::USB::Element::Magic)]->setText(tr("Magic"));
```

Comment:
This PR is a good excuse to add the using keyword, ie have using Element = IOS::HLE::USB::Element so that we can get rid of every other IOS::HLE::USB, same for Game.